### DISCUSSION TITLE
Release v1.5.2 / 5.60.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2412] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various issues and minor improvements.

**Note:** if you encounter the new message **SBIE2227**, the right course of action is to enable 8.3 name support for the volume you are using. Assuming you are using for example a RAM disk with the letter `R:\`, use the following command in an elevated command prompt:

```
fsutil behavior set disable8dot3 R: 0
```

This is not a new issue, but an old forgotten one. 
The message has been added to bring attention to potential problems and offer instructions how to properly resolve the issue.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Changed
- utility groups are now cleaned up automatically
- improved removal of leftovers [#2409](https://github.com/sandboxie-plus/Sandboxie/pull/2409)

### Fixed
- fixed issues with SBIE2227 on virtual drives
- fixed issues on native 32-bit windows introduced with 1.5.0 [#2401](https://github.com/sandboxie-plus/Sandboxie/issues/2401)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.2'>Release v1.5.2 / 5.60.2</a>.</em>

**Comment by lwcorp** [comment]
> Note: this installers are not signed as its an um scheduled hotfix release

Is this why it's flagged by an antivirus?
https://www.virustotal.com/gui/file/62c6c6dcfd51874d93bf559f0a6af1269068286dbf4d1b1d0d4d86712d789323

This is the first time in a long time I've seen a Sandboxie Plus release being flagged.

Could you [contact them](https://trapmine.com/company/contact-us/) to discuss this?

**Reply by isaak654** [reply]
It is unlikely that the lead developer would consider 1 detection out of 71.

Feel free to report it yourself according to the following suggestion:

> Trapmine E-mail Submission for False-Positives: support[at]trapmine.com and info[at]trapmine.com
>
> E-mail with attachment should be submitted in the below format:
 
 ```
To: support[at]trapmine.com
cc: info[at]trapmine.com
Subject: False Positive: file being detected by a Trapmine product
Email body text:

Could you please check the attached file, as I think it is a false detection. 

Description of issue: This file has been falsely detected as malware
```

Source: https://www.autohotkey.com/boards/viewtopic.php?t=62266

**Comment by bhafer** [comment]
Installer was blocked by Windows 10:
"Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk."

**Comment by DavidXanatos** [comment]
Well that's because of the missing signature, do you get the error with the installer or when trying to run the installed files, i would assume its only during instalation and there should be an option to bypass that.

**Reply by bhafer** [reply]
Thank you. Yes, it's running the installer.

**Comment by BilleBarrett** [comment]
I'm getting this, I just uninstalled Sandboxie-Plus 1.5.2 and reinstalled it, waiting to see if that fixes it
Bill
Sandboxie crashes my Chrome browser at least once per day. Same problem as last time with the same recovery message when I restart the machine "Sandboxie started in portable mode..."
Art
